docs/faq/frequently_asked_questions.rst

```diff
@@ -0,0 +1,57 @@
+    * We also have a supplementary application called :ref:`Vertiq Testing Tool <vertiq_testing_tool_guide>` to help you test your module. You can download and run Vertiq Testing Tool for more advanced testing of your module's configuration.
+
+#. Why does my module not spin when I send it a PWM signal?
+    * Vertiq modules require initial configuration before being able to spin with PWM commands. You can configure your motor using :ref:`Control Center <control_center_start_guide>`. The most commonly missed configuration when trying to spin with PWM is the :ref:`Motor Direction <throttle_direction>`, which must be set to something other than unconfigured. We recommend setting it to 2D Counter-Clockwise for your initial setup. You should also check what :ref:`Mode <throttle_mode>` is set on your module and that the corresponding :ref:`Maximum <throttle_maximums>`  for that mode is set. If you are still having trouble, refer to the :ref:`PWM and DSHOT tutorial <hobby_fc_tutorial>` or your module's Getting Started documentation: :ref:`Speed <speed_module_start_guide>`, :ref:`Servo <servo_module_start_guide>`, or :ref:`Pulsing <pulsing_module_start_guide>`.
+
+#. I’m trying to use DroneCAN but my motor isn’t spinning.
+    * A common issue that may cause your module not to spin is that its parameters are not configured correctly. Please confirm that you have properly configured the :ref:`Motor Direction <throttle_direction>`, :ref:`Mode <throttle_mode>`, and :ref:`Maximums <throttle_maximums>` for your module. Also, confirm that your configured :ref:`ESC Index <dronecan_px4_tutorial_esc_index>` is correct for the module that you are trying to control. If your module is not using :ref:`Arming Bypass <dronecan_arming_and_bypass>`, confirm that you are sending DroneCAN commands that will :ref:`arm <dronecan_arming_and_bypass>` the module initially. By default, commands close to 0% throttle should arm the module. For more information on how to use DroneCAN with your module, please refer to our :ref:`DroneCan Integration with PX4 and Ardupilot <dronecan_fc_tutorial>` documentation.
+
+#. What hardware do I need prior to receiving my module?
+    * Certain modules come with included hardware, such as prop adapters and heatshrink. Please refer to your module’s family page found in the sidebar under the Modules section for everything that is included in the box.
+    * You will need screws to attach a propeller to your module and to attach your module to the drone arm. Please check your module’s datasheet found on the module's family page under “Documentation” on our `website <https://www.vertiq.co/>`_. It is also highly recommended to apply a threadlocker, such as Loctite 243, to your screws when attaching a propeller to your module.
```

Comment:
semantics I know...and I'm honestly not sure of the answer, you should probably ask mike/wilson...do they need screws or bolts? I'd think bolts...but I'm not sure that it really even matters. My worry is that someone's going to buy a sharp, pointy screw, and tear open their phase wire which would be no good

Reply:
flat ended screws are still screws, they're just machine screws. Screws is fine to say here I think